UPtables/script/timetabling.lp

```diff
@@ -52,6 +52,10 @@ N { assigned(C,D,P) : d(D), ppd(P) } N :- course(C,_,N,_,_,_).
 
 :- not { assigned(C,D,P) : c(C) } N, d(D), ppd(P), rooms(N).
 
+%%% Room Unavailability: If a room is unavailable during a certain time, no
+%%% lecture be given in this room at that time.
+:- assigned(C,R,D,P), room_unavailable(R,D,P).
+
```

Comment:
please note: I was wrong, Mutsunori doesn't have the room unavailability in the original encoding. ```room_constraint``` means a room is unsuitable for a given course. Not for a given time. I had create the constraint myself.